test/js/lib/mlang.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('../../../src/lib/dao');
+require('../../../src/lib/mlang');
+
+function buildPersonDAO() {
```

Comment:
I strongly prefer setup code like this to just live in the beforeEachTest.

Helper methods in tests just obfuscate what exactly you are testing and make it harder to update tests if the helper method ever gets shared between two tests.